### ISSUE TITLE
Is the yml file connect with the website

### ISSUE THREAD

**Issue by HatemKhattab** [TechForPalestine/boycott-israeli-tech-companies-dataset#44]
If u visit https://www.israelitechalternatives.com/company/monday/ 
you cant see alternativ but there is data about that in the yml ?!

**Comment by abarmawi** [comment]
@HatemKhattab no, right now the website has its own dataset. There is a plan to sync both of them in the future.
